flink-rpc/flink-rpc-core/src/main/java/org/apache/flink/runtime/rpc/FencedRpcEndpoint.java

```diff
@@ -65,29 +54,14 @@ protected FencedRpcEndpoint(
                         mainThreadExecutable, this::validateRunsInMainThread, endpointId));
     }
 
-    protected FencedRpcEndpoint(RpcService rpcService, @Nullable F fencingToken) {
+    protected FencedRpcEndpoint(RpcService rpcService, @Nonnull F fencingToken) {
```

Comment:
We don't usually use `@Nonnull`, as anything not `@Nullable` is expected non-null.

Reply:
Get, will remove it